python/.envTemplate

```diff
@@ -3,10 +3,11 @@
 ###########################
 
 ## Used for logging in to Synapse
-# For security purposes, you should provide your apikey rather than
-# your password. Your apikey can be found on My Dashboard > Settings.
-SYN_USERNAME=username
-SYN_APIKEY="apikey"
+# For security purposes, you should provide your personal access token
+# (PAT) rather than your password. You can generate PATs on Synapse by
+# going to Account Settings > Manage Personal Access Tokens.
+export SYN_USERNAME=username
+export SYN_PAT="PAT"
```

Comment:
A cool feature is that the synapsePythonClient actually looks for the env variable `SYNAPSE_AUTH_TOKEN`.  So by doing:

```
export SYNAPSE_AUTH_TOKEN="PAT"
```

the client will auto login by just doing `synapseclient.login()`